### PR TITLE
Added port to description

### PR DESCRIPTION
**Pull request by ajayyy**
Looks great! Nice job

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <extension point="xbmc.service" library="service.py"/>
 
     <extension point="xbmc.addon.metadata">
-        <summary lang="en_GB">Skip YouTube in-video sponsors</summary>
+        <summary lang="en_GB">Skip YouTube in-video sponsors (Port)</summary>
```
Comment (siku2) [inline addon.xml]: I think this reads somewhat poorly because the summary doesn't mention the name "SponsorBlock".

File: addon.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
         <description lang="en_GB">
 SponsorBlock is a crowdsourced project to skip sponsor segments in YouTube videos.
```
Comment (siku2) [inline addon.xml]: How about we add it to the description?

```suggestion
This is a port of the SponsorBlock browser extension.

SponsorBlock is a crowdsourced project to skip sponsor segments in YouTube videos.
```

Comment (siku2) [reply addon.xml]: There's also the [`disclaimer`](https://kodi.wiki/view/Addon.xml#.3Cdisclaimer.3E) tag we could use:
```xml
<disclaimer lang="en_GB">This is an unofficial port</disclaimer>
```

**Comment by ajayyy** [comment]
That works

**Comment by siku2** [comment]
I updated the description and added a disclaimer to make it clear that this is a port.
The README now also mentions it.

Don't hesitate to contact me if there's anything else.

**Pull request closed without merging**